### PR TITLE
Signed-off-by: kiselitza <aldin@cerbos.dev>

### PR DESCRIPTION
**Pull request by kiselitza**
Enhanced aliases of pyprojectx tool config. README polished.

#### Description

Fixes #

#### Checklist 

- [x] The PR title has the correct prefix 
- [ ] PR is linked to the corresponding issue
- [x] All commits are signed-off (`git commit -s ...`) to provide the [DCO](https://developercertificate.org/)

### CODE REVIEW

**Comment by charithe** [comment]
This PR needs work. It shouldn't have been merged. Please re-create.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
-Run the following command to try out the demo:
+<img src="https://github.com/cerbos/cerbos/blob/main/docs/supplemental-ui/logo.png?raw=true" alt="Cerbos" width="87" height="100"/>
 
+This project `demo-python` will help you get familiar with using [Cerbos](https://github.com/cerbos/cerbos) in your Python applications.
+
+It's an app that follows access control regulation of a company employee `Harry` in process from submitting it, to getting his holiday request approved. Simply put, `demo-python` shows what are `Harry` and his work colleagues allowed to do with his `holiday request`.
+
+## Requirements
+- Minimum Python version supported by this demo is 3.10
+- Your Cerbos server is up and running (via either a [binary](https://docs.cerbos.dev/cerbos/latest/installation/binary.html), [container](https://docs.cerbos.dev/cerbos/latest/installation/container.html) or [helm](https://docs.cerbos.dev/cerbos/latest/installation/helm.html))
```
Comment (charithe) [inline README.md:11]: This is not necessary

File: README.md [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+
+⚠️ If you're completely new to Cerbos make sure to check the _[How it works](https://cerbos.dev/how-it-works)_ videos!  ⚠️
+
+
+## Run the demo
+
+Make sure that the `console` / `terminal` tab you're using is positioned in the root of this `demo-python` repository.
+You can do that by using the basic `cd` command which we will not be disecting now, but feel free to check how `cd` works in [Windows](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cd) and [Linux/Unix](https://www.javatpoint.com/linux-cd#:~:text=Linux%20cd%20command%20is%20used,commands%20in%20the%20Linux%20terminal) environments.
```
Comment (charithe) [inline README.md:19]: I don't think we need to go into this level of detail. Our target audience is technical users who are familiar with these basic things.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
+
+Now that you're properly positioned, simply run the following command:
```
Comment (charithe) [inline README.md:21]: Run the following command to launch the demo

File: README.md [hunk 4]
```diff
@@ -1,15 +1,41 @@
...
 The demo script (`main.py`) starts a new Cerbos container with the policies from the `policies` directory and sends requests for a set of different principals and resources to demonstrate how policy evaluation works.
 
-Try deleting the `condition` block attached to the `direct_manager` derived role (line 23--28 in `policies/derived_roles_1.yml`) and running `./pw demo` again. Amanda, who was previously disallowed from viewing or approving Harry’s leave requests should now be allowed to do those actions. 
+Now try deleting the `condition` block attached to the `direct_manager` derived role (line 23-28 in [derived_roles_1.yml](policies/derived_roles_1.yml)) and run the `./pw demo` command again.
+
+Amanda, who was previously disallowed from viewing or approving Harry’s leave requests should now be allowed to do those actions.
+
+Do you understand why?
+
+Reach out to [Cerbos Slack Community]() where we'll gladly put it in simple terms. There you get to ask pretty much everything cerbos-related you might think of.
```
Comment (charithe) [inline README.md:36]: Read more about Cerbos on https://docs.cerbos.dev or join the friendly [Cerbos Slack Community](https://community.cerbos.dev) to ask questions.

File: README.md [hunk 5]
```diff
@@ -1,15 +1,41 @@
...
 
+**P.S.** Feel free to get creative and edit [policies](policies) to test how even more complex use-cases would turn out.
```
Comment (charithe) [inline README.md:38]: Remove "P.S"

File: README.md (hunk 2 above)
Comment (kiselitza) [reply README.md:19]: Hmm, I agree with you on this one. I had plenty of experiences where people were asking silly things but it takes more space than it adds value, will remove it.

**Pull request merged**